src/components/Match/matchColumns.jsx

```diff
@@ -924,13 +924,78 @@ export const objectiveDamageColumns = [heroTdColumn].concat(Object.keys(strings)
   relativeBars: true,
 })));
 
+const dmgTargetValueStyle = {
+  position: 'absolute',
+  left: '43px',
+  width: '30px',
+  height: '10px',
+  bottom: '33px',
+  fontSize: '10px',
+  textAlign: 'center',
+  lineHeight: '0.9',
+  fontWeight: constants.fontWeightMedium,
+  verticalAlign: 'center',
+  zIndex: '1',
+  backgroundColor: constants.darkPrimaryColor,
+};
+
+const dmgTargetIconStyle = {
+  height: '30px',
+  bottom: '30px',
+  left: '25px',
+  marginLeft: '8px',
+  position: 'relative',
+  backgroundColor: 'rgba(255, 255, 255, 0.1)',
+};
+
+const damageTargetIcons = (t) => {
+  const targets = [];
+  Object.keys(t).forEach((target) => {
+    const hero = getHeroesById()[target];
+    const heroicon = heroes[hero.id] && process.env.REACT_APP_API_HOST + heroes[hero.id].icon;
+    const j = (
+      <div style={{ float: 'left', position: 'relative', paddingLeft: '10px' }}>
+        <span style={dmgTargetValueStyle}>{`${t[target]}`}</span>
+        <img
+          src={heroicon}
+          alt=""
+          style={dmgTargetIconStyle}
+          data-tip={`${hero.localized_name} ${t[target]}`}
+          data-offset="{'right': 5}"
+          data-delay-show="50"
+        />
+        <ReactTooltip place="top" effect="solid" />
```

Comment:
I don't think the number on the tooltip is necessary since it's already on the icon